cmd/osv-scanner/fixtures/sbom-insecure/alpine.cdx.xml

```diff
@@ -601,4 +601,4 @@
       </properties>
     </component>
   </components>
-</bom>
```

Comment:
is this intentional?

Reply:
I think the new line got added because of vscode formatting.